docs/examples/basic_conceptual_guide/Part_3-model_analyzer/Readme.md

```diff
@@ -0,0 +1,108 @@
+
+```
+sudo apt-get update && sudo apt-get install python3-pip
+sudo apt-get update && sudo apt-get install wkhtmltopdf
+pip3 install triton-model-analyzer
+```
+
+### Usage details
+
+Before diving into details with an example, a discussion about the overall functionality and capability is needed to understand how best to use the Model Analyzer tool. Let's begin the discussion by focusing on what is most important to the users: setting `objectives` and `constraints` for the sweeps.
+
+- **objectives**: Users can choose to order the results on the basis of their deployment goals, throughput, latency, or tailoring to specific resource constraints. [Learn more](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config.md#objective).
+    
+    Model Analyzer has two modes, Online and Offline. In online mode users can specify latency budgets for their deployments to cater to their requirements. For Offline mode a similar specification can be mode for minimum throughput. [Learn more](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/cli.md#model-analyze-modes)
+
+- **constraints**: Users can also choose to constrain the selection of sweeps to specific requirements for throughput, latency or gpu memory utilization. [Learn more](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config.md#constraint)
+
+With the more broader selections discussed, let's talk about the three key sub commands that users need to use to make use of Model Analyzer: `profile`, `analyze`, and `report`. While most of the settings for these commands can be set using a flag, some require building a configuration file. Refer to [this section](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config.md) of the documentation for a comprehensive list of settings.
+
+- **profile**: `profile` is used to run the benchmarking sweeps. This is where users specify the sweep space details like number of instances per GPU, range of maximum batch sizes for the model, maximum cpu utilization, batch sizes of the queries sent, number of concurrent queries sent to Triton and [more](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config.md#config-options-for-profile). `profile` runs these sweeps, records the performance of each of the configurations and saves runs are checkpoints. Think of this step as simply running a large number of experiments and recording data points for analysis. This step will take 60-90 minutes to run. Refer to the [documentation](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config.md#config-options-for-profile) for more information or if you want a quicker and smaller sweep. 
```

Comment:
We should be using quick search as our example/common case and encouraging users not to set limits/ranges for batch size/instance count unless absolutely necessary. Quick search has a typical runtime of around 10-20 minutes.